tools/ci_build/github/windows/bundle_dlls_gpu.bat

```diff
@@ -3,20 +3,20 @@ REM Licensed under the MIT License.
 
 REM for available runtime identifiers, see https://github.com/dotnet/corefx/blob/release/3.1/pkg/Microsoft.NETCore.Platforms/runtime.json
```

Comment:
How about this: we make this file solely for the new jobs you are adding? And remove the nuget part in this file.

Reply:
Yes, I can remove this link.